DROD/PendingPlotEffect.cpp

```diff
@@ -142,7 +175,7 @@ void CPendingPlotEffect::PlotSwordsman(
 //
 //Params:
 	const UINT wStartX, const UINT wStartY, const UINT wEndX, const UINT wEndY,
-	SDL_Surface* pDestSurface)
+	SDL_Surface& pDestSurface)
```

Comment:
Probably should rename pDestSurface --> destSurface wherever the type changed from a pointer to a reference.

Reply:
You asked for it - 73 files updated :).

Reply:
Thank you. I really appreciate the care in making it easy to read!